security/security.service.ts

```diff
@@ -0,0 +1,199 @@
+import { getEnv } from '@helper/environment';
+import { IconikToken, IconikTokenModel, IconikTokenSchema } from '@models/DynamoDB/iconik-token.model';
+import { IconikService } from '@workflowwin/iconik-api';
+import { CustomActionSchema } from '@workflowwin/iconik-api/dist/src/assets/assets-methods';
+import { WebhookResponseSchema } from '@workflowwin/iconik-api/src/notifications/notifications-methods';
+import { CloudWatchEvents, Lambda } from 'aws-sdk';
+import * as AWS from 'aws-sdk';
+import {
+  DescribeRuleResponse,
+  PutRuleRequest,
+  PutRuleResponse,
+  PutTargetsRequest,
+  PutTargetsResponse,
+} from 'aws-sdk/clients/cloudwatchevents';
+import { AddPermissionResponse } from 'aws-sdk/clients/lambda';
+import { ScanResponse } from 'dynamoose/dist/DocumentRetriever';
+
+export class SecurityService {
+  private readonly cloudWatchEvents: CloudWatchEvents;
+  private readonly lambda: Lambda;
+
+  constructor() {
+    this.cloudWatchEvents = new AWS.CloudWatchEvents({ apiVersion: '2015-10-07' });
+    this.lambda = new AWS.Lambda();
+    AWS.config.update({ region: getEnv('REGION') });

```

Comment:
Why do you call it here? What problem do you try to solve?

Reply:
![image](https://user-images.githubusercontent.com/62546525/119312332-e8a39a00-bc7a-11eb-94ea-df05059db92e.png)

Example from the documentation.

Reply:
I believe it should work without setting a region

Reply:
I believe too. Because AWS SDK should get region for us. And if not then it is not a best place to set GLOBAL region.